src/System.IO.Pipelines.Networking.Libuv/UvTcpConnection.cs

```diff
@@ -24,6 +22,7 @@ public class UvTcpConnection : IPipeConnection
 
         private Task _sendingTask;
         private WritableBuffer? _inputBuffer;
+        private MemoryHandle? _inputBufferPin;
```

Comment:
I was going to get around to asking if we should make `Free` idempotent. I've explicitly always set it to `null` in the unpin method because it's not safe to call multiple times. So that rules out the possibility of that happening